src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -475,6 +489,15 @@ public String getByteLengthFunctionName(TypeRef typeRef) {
     return getNotImplementedString("SurfaceNamer.getByteLengthFunctionName");
   }
 
+  /** The name of the class that ties in the helper functions to a client partial veneers. */
+  public String getHelpersClassName(String apiShortName) {
+    return publicClassName(Name.anyCamel(apiShortName).join("helpers"));
+  }
+
+  /** The name of a stub type of an interface */
+  public String getStubType(Interface apiInterface) {
```

Comment:
It's no different. The full stub type was stored in the property that held grpcClientTypeName. Which didn't seem correct. This was to fix that so that I could use the namer.getGrpcClientTypeName for the grpc client type name.